source/common/config/config_provider_impl.h

```diff
@@ -212,16 +238,12 @@ class ConfigSubscriptionCommonBase : protected Logger::Loggable<Logger::Id::conf
   }
 
   const std::string name_;
-  std::unordered_set<MutableConfigProviderCommonBase*> mutable_config_providers_;
   absl::optional<LastConfigInfo> config_info_;
+  // This slot holds a Config implementation in each thread, which is shared between
+  // bound providers.
```

Comment:
nit: there are no explicitly bound providers anymore. It would be clearer to specify that all ConfigProviders for the same config source share this.

Reply:
done